ci-operator/step-registry/azure/provision/service-principal/minimal-permission/azure-provision-service-principal-minimal-permission-commands.sh

```diff
@@ -118,9 +118,6 @@ az --version
 
 # set the parameters we'll need as env vars
 AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
-if [[ -f "${CLUSTER_PROFILE_DIR}/installer-sp-minter.json" ]]; then
```

Comment:
also [over here][1], deprovision-side.

[1]: https://github.com/openshift/release/blob/5f4835e11beff590a250d1d33abedc0c721bcbff/ci-operator/step-registry/azure/provision/service-principal/minimal-permission/azure-provision-service-principal-minimal-permission-commands.sh#L122

Reply:
oops, [this|https://github.com/openshift/release/blob/6bfd7b6b8651ad48a8f9aea680f9d0a830d8e5a3/ci-operator/step-registry/azure/deprovision/sp-and-custom-role/azure-deprovision-sp-and-custom-role-commands.sh#L21] is the deprovision side.